proto/minder/v1/minder.proto

```diff
@@ -1987,12 +2028,57 @@ message RoleAssignment {
     string role = 1;
     // subject is the subject to which the role is assigned.
     string subject = 2;
+    // display_name is the display name of the subject.
+    string display_name = 5;
     // projectt is the projectt in which the role is assigned.
     optional string project = 4;
 
     reserved 3; // deprecated context
 }
 
+message ListInvitationsRequest {
+}
+
+message ListInvitationsResponse {
+    repeated Invitation invitations = 1;
+}
+
+message ResolveInvitationRequest {
+    // nonce is the nonce of the invitation to resolve.
+    string nonce = 1;
+    // accept is true if the invitation is accepted, false if it is rejected.
+    bool accept = 2;
```

Comment:
Honest question: Could we ever have another action? I wonder if the bool would bite us if we ever need another action like e.g. `view` (but I guess we can add another field then slowly deprecate this one)

Reply:
I thought about this.  😁  I'm generally suspicious of boolean fields, so I don't feel strongly about keeping this, but I couldn't construct a meaningful difference between a bool and an enum, so I opted for simplicity.

This RPC is about retiring an invitation.  It _feels_ like `accept` or `not accept` are the two options that cause the invitation to be removed.  I can imagine a "delegate" or "forward" action in the future, but I think I would express those with `accept = false` and a new field indicating the new recipient of the invitation.

I welcome @kantord sharing perspective about how using a boolean here (with default false) would affect the UI implementation.